zOSMF-Samples/workflow_sample_wf2wf.xml

```diff
@@ -0,0 +1,203 @@
+		Here the called workflow will be workflow_sample_calledwfBasic.xml.
+		If a instance with the specified calledWorkflowID already exist, workflow will directly open this instance when user start perform it, if the instance does not exist or multiple instances
+		with the calledWorkflowID exist, a dialog will pop up for user to create a new called workflow instance.
+	 -->
+<step name="simpleWf2wf">
+	<title>A very simple sample of workflow to workflow execution.</title>
+	<description>This is a very simple demo of using workflow to worfklow execution in which only the one mondatory element named calledWorkflowID is specified.</description>
+	<callingStepWeight>2</callingStepWeight>
+	<calledWorkflowDescription>The workflow definition file more likely locates under directory: /usr/lpp/zosmf/samples.</calledWorkflowDescription>
+	<calledWorkflowID>workflow.sample.calledwfBasic</calledWorkflowID>
+</step>
+ 
+<!-- This step is used for user to input below variable values some of which will be used to transfer to the following called workflow:
+		String1
+		String2
+		sampleString
+ -->
+<step name="prepareVariables" >
+	<title>Prepare Variable Step</title>
+	<description>In this step, you enter some variable values to see if they can be transferred to the called workflow in the following step.</description>
+	<variableValue name="String1"/>
+	<variableValue name="String2"/>
+	<variableValue name="sampleString"/>
+	<instructions>User can input some values for the variables in this step, those values can be transfered to the called workflow if they satisfy the mapping rules that user specified in the following step.</instructions>
+	<weight>1</weight>
+	<skills>System Programmer</skills>
+</step>
+ 
+<!-- This step demonstrates the use of variableMapping element for user to transfer variable values from calling workflow to called workflow.
+		When the called workflow instance is created successfully, variables that satisfy either the regular expression or the variableName
+		mapping will be transferred from calling workflow to called workflow.  In this case, below variables satisfy the <regExpression> and their
+		values will be transferred from calling workflow to called workflow:
+			- String1
+			- String2
+		Additionally, <variableName> element will map the variable "sampleString" in this calling workflow to the variable "demoString" in the called workflow
+		and transfer its value to demoString in called workflow.
+		
+		Note: When user specify the XML definition and create the called workflow successfully, the variables that meet the mapping rules will be transferred
+		from the calling workflow to the called workflow only once, that means, no matter how many times the user perform the calling step in future, those 
+		variable values will not be transferred any more. But if user decide to change the existing called workflow, this will be treated as creating a new called
```

Comment:
```suggestion
		variable values will not be updated. But if user decide to change the existing called workflow, this will be treated as creating a new called
```